### DISCUSSION TITLE
One AM instance for several teams, managing silences

### DISCUSSION THREAD

**Discussion by grzesuav** [prometheus/alertmanager#2963] [Ideas]
Hi,

I know that prometheus do not handle user access by default, however thing can be messy with Alertmanager, i.e. one team can accidentally silence alert of other team. From other side, forbidding the teams to use silence at all also is not desired. Wanted to start discussion how it can be handled.

I do not have solution for that (other than setting up AM per team, which is a overhead in bigger organization)

**Comment by TheMeier** [comment]
related https://github.com/prometheus/alertmanager/issues/4959
